lib/remote_helper.py

```diff
@@ -313,8 +314,14 @@ def enable_root_login(cls, host, keyname, infrastructure, is_verbose):
       # logged-in user, so log in as that user to enable root login.
       if infrastructure == "gce":
         AppScaleLogger.log("Root login not enabled - enabling it now.")
-        cls.ssh(host, keyname, 'sudo cp ~/.ssh/authorized_keys /root/.ssh/',
+        temp_file = tempfile.NamedTemporaryFile()
```

Comment:
You probably don't need `tempfile` here since you don't need to make a temporary file on the machine running the tools. Maybe use `mktmp` as part of the ssh command.